modules/rest-http/src/main/java/org/apache/ignite/internal/processors/rest/protocols/http/jetty/GridJettyRestHandler.java

```diff
@@ -759,6 +761,20 @@ else if (cmd == CLUSTER_ACTIVE || cmd == CLUSTER_ACTIVATE)
                 break;
             }
 
+            case BASELINE_CURRENT_STATE:
+            case BASELINE_SET:
+            case BASELINE_ADD:
+            case BASELINE_REMOVE: {
+                GridRestBaselineRequest restReq0 = new GridRestBaselineRequest();
+
+                restReq0.topologyVersion(longValue("topVer", params, null));
+                restReq0.consistentIds(values(null, "consistentId", params));
```

Comment:
This looks confusing. Param named "consistentId" is actually the collection of ids. I propose to rename it.

Reply:
I believe that you actually used "consistentIds" in other places, like in missingParameter("consistentIds").

Reply:
"consistent If" it's not a parameter name, its parameter prefix for passing multiple values through REST. Look at the "values" method.

Reply:
Now I see, thank you!